api/src/main/java/org/openmrs/module/metrics/MetricEvent.java

```diff
@@ -17,7 +17,7 @@
 	
 	private LocalDateTime timeStamp;
 	
-	private URI uri;
+	private String object_uuid;
```

Comment:
Prefer names like `objectUuid` over `object_uuid` or, in other words, in Java we tend to prefer `camelCase` names.